### PR TITLE
doc: concepts / hooks / sidebar

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

File: docs/content/workflows/design/hooks/webhook.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 title = "Webhook"
-weight = 3
+weight = 2
 
 +++
 
-TODO
\ No newline at end of file
+On a Root Pipeline, you can add a "Webhook". Click on the created icon to get the WebHook URL.
+
```
Comment (bnjjj) [inline docs/content/workflows/design/hooks/webhook.md]: In order to trigger this one you just have to make a curl on the given url with the selected method

File: docs/content/workflows/design/sidebar.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
++++
+title = "Sidebar"
+weight = 8
+
++++
+
+The "select box" to select a git branch or a version on UI has been removed. There is only one select box for filter on CDS Tags.
```
Comment (bnjjj) [inline docs/content/workflows/design/sidebar.md]: is it useful for newcomers to talk about previous select box ? Maybe an example is better.

For example if you want to know on which branch the build was launched you just have to filter on a specific CDS tag (in this case "git.branch")

File: docs/content/workflows/design/sidebar.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Inside a job, a user can add a Tag with the worker command 
+
+```
+worker tag tagKey=tagValue
+```
+
+See [worker tab documentation]({{< relref "worker/commands/tag.md" >}})
+
+Tags are useful to add indication on the sidebar about the context of a Run.
```
Comment (bnjjj) [inline docs/content/workflows/design/sidebar.md]: Tags are useful to add informations and context for a run

File: docs/content/workflows/design/sidebar.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+You can select the tags displayed on the sidebar Workflow → Advanced → "Tags to display in the sidebar".
```
Comment (bnjjj) [inline docs/content/workflows/design/sidebar.md]: + if you want to filter all runs in sidebar

File: docs/content/hosting/repositories-manager/_index.md [hunk 5]
```diff
@@ -4,12 +4,11 @@ weight = 7
 
 +++
 
-
 CDS can be linked to following repositories manager :
 
- - **Atlassian Stash / Bitbucket**
- - **Github**
- - **Gitlab**
+ - **[Atlassian Stash / Bitbucket]({{< relref "bitbucket.md" >}})**
```
Comment (fsamin) [inline docs/content/hosting/repositories-manager/_index.md]: Bitbucket Server

File: docs/content/introduction/cli/_index.md [hunk 6]
```diff
@@ -4,5 +4,45 @@ weight = 4
 
 +++
 
-TODO
 
+## Download
+
+You'll find last release of `cdsctl` on [Github Releases](https://github.com/ovh/cds/releases/latest).
+
+## Commands
```
Comment (fsamin) [inline docs/content/introduction/cli/_index.md]: it will be painful if we had to copy and paste the cdsctl help every time we change it

Comment (yesnault) [reply docs/content/introduction/cli/_index.md]: this will be auto-generated, next PR

**Pull request merged**